gradle/libs.versions.toml

```diff
@@ -15,16 +15,15 @@
 ##
 [versions]
 accessibilityTestFramework = "4.0.0"
-accompanist = "0.28.0"
 activityCompose = "1.7.0-alpha03"
 androidGradlePlugin = "7.3.1"
 appcompat = "1.6.0"
 benchmark = "1.1.0-rc02"
 # @keep
 compileSdk = "33"
 composeLatest = "1.4.0-alpha03"
-composeBom = "2023.01.00"
-compose-compiler = "1.3.2"
+composeBom = "2023.04.00"
```

Comment:
Nit: we can use the latest version of Compose BOM here but let's do that in a separate PR.